src/EncompassRest/Loans/Enums/ExeClosDocToBeRtrndHrs.cs

```diff
@@ -0,0 +1,12 @@
+using System.Runtime.Serialization;
+
+namespace EncompassRest.Loans.Enums
+{
+    public enum ExeClosDocToBeRtrndHrs
+    {
+        [EnumMember(Value = "24")]
```

Comment:
numeric values to match strings (eg. n24 = 24)

Reply:
In case you didn't see I've done as you suggested.